package.json

```
@@ -29,8 +29,9 @@
     "test": "npm-run-all test:assert:strict test:unit lint",
     "test:assert": "vows test/assert.vows.checks.js",
     "test:assert:strict": "XMLDOM_ASSERT_STRICT=1 npm run test:assert",
-    "test:unit": "npm run test:vows",
-    "test:vows": "vows 'test/**/*.vows.js'"
+    "test:unit": "npm-run-all test:vows jest",
+    "test:vows": "vows 'test/**/*.vows.js'",
+    "jest": "jest"
```

```suggestion
    "test:unit": "npm-run-all test:vows test:jest",
    "test:vows": "vows 'test/**/*.vows.js'",
    "test:jest": "jest"
```

I would normally rather keep it sorted alphabetically, but maybe it is good to keep more general entries before more specific entries.